storybookjs/storybook
# Ondevice actions

Issue:

## What I did

## How to test

- Is this testable with Jest or Chromatic screenshots? - maybe
- Does this need a new example in the kitchen sink apps? - I've added it to crna
- Does this need an update to the documentation? - I've updated ADDONS_SUPPORT

If your answer is yes to any of these, please make sure to include it in your PR.




It looks like this:

![Screenshot 2019-04-23 at 14 10 59](https://user-images.githubusercontent.com/1260646/56584178-254fc380-65d3-11e9-860f-2900a21073d4.png)
![Screenshot 2019-04-23 at 14 11 29](https://user-images.githubusercontent.com/1260646/56584187-2a147780-65d3-11e9-8199-7d2101bde94d.png)
